src/controllers/record.rs

```diff
@@ -13,13 +13,13 @@ pub(crate) struct RecordController;
 
 impl RecordController {
     pub async fn get_all(
-        Extension(postgres_service): Extension<Arc<PostgresService>>,
+        Extension(postgres_service): Extension<Arc<RecordService>>,
```

Comment:
I think RecordService has a Repository which is a Connection pool which uses an arc under the hood... So It's probably send + sync already, if that s a pgpool i think Clones are cheap, gotta check!